lib/model.js

```diff
@@ -1079,7 +1078,15 @@ Model.exists = function exists(filter, options, callback) {
     });
     return;
   }
-
+  console.log(options);
+  if (typeof options !== 'undefined') {
```

Comment:
This should be `if (options != null)`, because otherwise if `options` is `null` this will error out.

A better alternative would be to do `options = options || {};` so you don't even need this `if` statement. That ensures that `options` is an object.